src/main/java/com/example/GymProject/config/MyWebAppInitializer.java

```diff
@@ -0,0 +1,23 @@
+package com.example.GymProject.config;
+
+import jakarta.servlet.ServletContext;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.ServletRegistration;
+import org.springframework.web.WebApplicationInitializer;
+import org.springframework.web.context.support.AnnotationConfigWebApplicationContext;
+import org.springframework.web.servlet.DispatcherServlet;
+
+public class MyWebAppInitializer implements WebApplicationInitializer {
+
+    @Override
+    public void onStartup(ServletContext servletContext) throws ServletException {
+        AnnotationConfigWebApplicationContext context = new AnnotationConfigWebApplicationContext();
+        context.register(AppConfig.class);
+        context.setServletContext(servletContext);
+
+        ServletRegistration.Dynamic dispatcher = servletContext.addServlet("dispatcher", new DispatcherServlet(context));
+        dispatcher.setLoadOnStartup(1);
+        dispatcher.addMapping("/");
```

Comment:
Do we really need these servlets in current task? We do not have any RestController as far as I know. And servlets are for http. The second thing is that I cannot understand, why do you create a new ApplicationContext in this class. You ApplicationContext is being created in the main app class. Let's think about it and find a way to get this context from Spring.